ui/js/component/publishForm/internal/channelSection.jsx

```diff
@@ -110,7 +110,7 @@ class ChannelSection extends React.PureComponent {
       </FormRow>
     );
     if (fetchingChannels) {
-      channelContent.push(
+      channelContent = (
```

Comment:
It's probably would have been fine to keep this as an array, switching the type of a variable via assignment is bad practice IMO.